src/hooks/index.ts

```diff
@@ -15,4 +15,22 @@ const useQueryStringAndParam = <T>() => {
   }, [params]);
 };
 
-export default useQueryStringAndParam;
+const useModal = () => {
+  const [isModalVisible, setIsModalVisible] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isModalVisible, setIsModalVisible] = useState(false);
```

요렇게 해도 타입추론 될 것 같슴다!

Reply:
앗 commit suggestion할걸...😅 `a025ec0`에 반영했습니닷!